study.md

```diff
@@ -21,7 +21,7 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+We use 'this' as a shortcut to an object. http://javascriptissexy.com/understand-javascripts-this-with-clarity-and-master-it/.
 ```
```

Comment:
I haven't heard it described that way before but I like that. Be sure to consider why it's a shortcut and remember that one reason we might use "this" is to avoid ambiguity by making sure we are
correctly referring to the current object, instead of perhaps another globally
defined variable with the same name as the object.